buildchain/buildchain/targets/package.py

```diff
@@ -143,10 +143,36 @@ def make_package_directory(self) -> types.TaskDict:
 
     def generate_meta(self) -> types.TaskDict:
         """Generate the .meta file for the package."""
+        container_spec = '/rpmbuild/SPECS/{}'.format(self.spec.name)
+        container_meta = '/rpmbuild/META/{}'.format(self.spec.name)
+        mounts = [
+            DockerRun.ENTRYPOINT_MOUNT,
+            DockerRun.bind_ro_mount(source=self.spec, target=container_spec),
+            DockerRun.bind_mount(
+                source=self.meta.parent, target=container_meta
+            )
+        ]
+        command = ['/entrypoint.sh', 'buildmeta']
+        rpmspec_config = {
+            'hostname': 'build',
+            'read_only': True,
+            'remove': True
+        }
+        buildmeta_callable = DockerRun(
```

Comment:
Could be simply called `buildmeta` no?

Reply:
To me this naming (and all subsequent variations thereof) are clearer when you're supplying the actions to the `doit` task dict, otherwise I'd keep it short and sweet.

Reply:
To me it was looking like hungarian notation (encoding the type in the name), which is weird in Python.